### PR TITLE
docs: Jira integration

### PR DESCRIPTION
**Pull request by dabeeeenster**
Beta is ready - will create a new PR when fully final Marketplace listing is ready.

### CODE REVIEW

File: docs/docs/integrations/project-management/jira.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+:::tip
+
+The Jira integration is currently in beta. Please contact us to join the beta!
+
+:::
+
+## Integration Setup
+
+todo: Correct the link
```
Comment (matthewelwell) [inline docs/docs/integrations/project-management/jira.md]: We should probably remove this todo (or add it as an actual md comment if that's possible?)

File: docs/docs/integrations/project-management/jira.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+1. Add the app from the
+   [Atlassian Marketplace](https://developer.atlassian.com/console/install/3fd8c838-2ced-45a5-8807-0401ec45a096?signature=5f47a1c11336d3ecd75054fbd534d808e5b22dd98afe47ceacee5ea6918426bc19bb2c2f7e740f2fed79d1d96b5b1fd007a088a684e85681fada20617e227083&product=jira).
+2. You need to provide your Flagsmith API key to Jira. You can get your API key by going to Account > Keys in Flagsmith.
```
Comment (matthewelwell) [inline docs/docs/integrations/project-management/jira.md:19]: Hmm... this is interesting! It reminds me that we need to check which API key it's using and make sure it's migrated to using the proper scoped API keys before we release. For now (in Beta) I guess it's fine to ask people to change their API key once it's released.

Comment (dabeeeenster) [reply docs/docs/integrations/project-management/jira.md:19]: Yep agreed.

File: docs/docs/integrations/project-management/jira.md (hunk 1 above)
Comment (dabeeeenster) [reply docs/docs/integrations/project-management/jira.md]: OK removed for now.

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**